cmd/thriftlint/lint.go

```diff
@@ -0,0 +1,193 @@
+package main
```

Comment:
what do you think of moving most of the logic into a library instead (APIs do not have to be guaranteed, but allows tools like `yab` to run the same lint rules)